build/vllm/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# vLLM requires GPU.
+FROM --platform=$BUILDPLATFORM vllm/vllm-openai:v0.6.2 as vllm
+ARG TARGETARCH
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip install librosa soundfile
```

Comment:
We probably can file an issue or PR to `vllm` to upstream `Dockerfile` changes? @kkaneda

Reply:
sounds good!